src/components/Dashboard/StaffNotation.vue

```diff
@@ -128,6 +136,13 @@
                 </v-list-item-title>
               </v-list-item>
             </template>
+            <template v-if="!isHistorical">
+              <v-list-item @click="showAdministerFreezeDialog()">
```

Comment:
Please format this line like the others above and add the data-type property.

Also please update the unit tests for this component.